apps/api/tsconfig.app.json

```diff
@@ -5,6 +5,16 @@
     "module": "commonjs",
     "types": ["node"]
   },
-  "exclude": ["jest.config.ts", "src/**/*.spec.ts", "src/**/*.test.ts"],
+  "exclude": [
+    "vitest.config.ts",
+    "src/**/*.spec.ts",
+    "src/**/*.test.ts",
+    "src/**/*.test.tsx",
+    "src/**/*.spec.tsx",
+    "src/**/*.test.js",
+    "src/**/*.spec.js",
+    "src/**/*.test.jsx",
+    "src/**/*.spec.jsx"
```

Comment:
I don't think we need all these extra extensions.

Same goes for `tsconfig.app.json`s and `tsconfig.spec.json`s in other apps. The `website` project should also include `.tsx` extensions in addition to `.ts`, but that's it.